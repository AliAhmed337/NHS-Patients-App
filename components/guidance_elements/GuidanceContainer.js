import React from 'react';
import { FontAwesome } from '@expo/vector-icons';
import { View, Text } from 'react-native';

export const GuidanceContainer = (props) => {
	return (
		<View>
			<View style={{ flexDirection: 'row', alignItems: 'center' }}>
				<FontAwesome name="bullseye" size={22} color="blue" />
				<Text style={styles.titleContainterStyle}>{props.title}</Text>
			</View>

			<View >
				<Text style={styles.contentContainerStyle}>{props.content}</Text>
			</View>
		</View>
	);
};

const styles = {
	titleContainterStyle: {
		margin: 6,
		fontSize: 22,
		fontWeight: 'bold'

	},
	contentContainerStyle: {
		margin: 6,
		fontSize: 18
	}
};