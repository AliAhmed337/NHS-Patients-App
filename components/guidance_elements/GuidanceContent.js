import React from 'react';
import { View, Text } from 'react-native';

export const GuidanceContent = (props) => {
		return (
			<View>
				<Text style={styles.contentStyle}>
					{props.content}
				</Text>
			</View>
		);
};

const styles = {
	contentStyle: {
		fontSize: 16,
	}
};