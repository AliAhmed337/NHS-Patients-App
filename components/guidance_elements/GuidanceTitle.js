import React from 'react';
import { View, Text } from 'react-native';

export const GuidanceTitle = (props) => {
	return (
		<View>
			<Text style={styles.titleStyle}>
				{props.title}
			</Text>
		</View>
	);
};

const styles = {
	titleStyle: {
		fontSize: 18,
		fontWeight: 'bold'
	}
};