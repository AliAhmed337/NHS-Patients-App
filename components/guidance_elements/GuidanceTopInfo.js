import React from 'react';
import { View, Text } from 'react-native';

export const GuidanceTopInfo = () => {
	return (
		<View>
            <Text style={styles.infoStyle}>
				Dear Patient, 
            </Text>

            <Text style={styles.infoStyle}>
            	We kindly ask you to follow these important instructions before your appointment:
            </Text>
        </View>
	);
};

const styles = {
	infoStyle: {
		fontSize: 18,
		textAlign: 'center',
		padding: 5
	}
};