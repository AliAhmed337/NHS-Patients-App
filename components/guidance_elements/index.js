export * from './GuidanceTopInfo';
export * from './GuidanceTitle';
export * from './GuidanceContent';
export * from './GuidanceContainer';