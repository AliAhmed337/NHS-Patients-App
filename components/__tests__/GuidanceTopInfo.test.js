import React from 'react';
import { shallow } from 'enzyme';
import { GuidanceTopInfo } from '../guidance_elements';

describe('Testing guidance info component', () => {

  it('renders correctly', () => {
    const wrapper = shallow(<GuidanceTopInfo />);
    expect(wrapper).toMatchSnapshot();
  });
});
