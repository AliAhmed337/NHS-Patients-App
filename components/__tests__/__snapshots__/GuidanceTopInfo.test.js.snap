// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing guidance info component renders correctly 1`] = `
<Component>
  <Component
    style={
      Object {
        "color": "#00315F",
        "fontFamily": "Arial",
        "fontSize": 16,
        "padding": 5,
      }
    }
  >
    Dear Patient,
  </Component>
  <Component
    style={
      Object {
        "color": "#00315F",
        "fontFamily": "Arial",
        "fontSize": 16,
        "padding": 5,
      }
    }
  >
    We kindly ask you to follow these important instructions before your appointment:
  </Component>
</Component>
`;
